index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+
+
+</head>
+
+
+<body>
+    <!--header-->
+    <!--nav menu-->
+    <div class="nav-bar ">
+        <div class="logo">
+             <a href="#" class="mylogo">CHESTER</a> </div>
+        <button><img src=" /images/Union.png " alt=" " class="hamburger "> </button>
+        <!--closing nav-->
+    </div>
+
+    <article>
```

Comment:
- [ ] **Required:** You have done well by using an `article` tag. However, you are required to use a `section` tag.